lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1440,19 +1441,25 @@ def test_progress_queries_paced_courses(self, self_paced, self_paced_enabled):
         """Test that query counts remain the same for self-paced and instructor-paced courses."""
         SelfPacedConfiguration(enabled=self_paced_enabled).save()
         self.setup_course(self_paced=self_paced)
-        with self.assertNumQueries(41), check_mongo_calls(4):
+        with self.assertNumQueries(42), check_mongo_calls(2):
             self._get_progress_page()
 
-    def test_progress_queries(self):
+    @ddt.data(
+        (False, 41, 27),
+        (True, 41, 27)
```

Comment:
These numbers don't seem right, but they pass for some reason... I'll keep digging